### PR TITLE
:book: Fix ironic installation with kustomize instruction

### LINKED ISSUE #298: User-guide: Update/fix Ironic installation guide
The instructions on the [Install Ironic](https://metal3io.netlify.app/ironic/ironic_installation.html) page does not work at least for the in-cluster installation. We have recently made changes to the kustomization, which also affects this, but even when checking out an older commit/release the commands are not working.

```console
❯ kustomize build ironic-deployment/basic-auth                                                        
Error: unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization' in directory '/home/user/workspace/baremetal-operator/ironic-deployment/basic-auth'
                                                                                                     
❯ kustomize build ironic-deployment/basic-auth/tls
Error: unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization' in directory '/home/user/workspace/baremetal-operator/ironic-deployment/basic-auth/tls'
```

### PR DESCRIPTION
**Pull request by mquhuy**
Fix #298

### CODE REVIEW

**Review by lentzi90 — COMMENTED** [review]
The content looks good, but the lines are very long.
I guess we don't enforce any line length here yet @tuminoid ?
Could you anyway perhaps wrap the lines so it is easier to read the "code" @mquhuy ?

**Comment by mquhuy** [comment]
> The content looks good, but the lines are very long. I guess we don't enforce any line length here yet @tuminoid ? Could you anyway perhaps wrap the lines so it is easier to read the "code" @mquhuy ?

Thanks @lentzi90. I've updated it.

File: docs/user-guide/src/ironic/ironic_installation.md [hunk 1]
```diff
@@ -62,6 +62,10 @@ prevent eavesdropping.
 
 ### Installing with Kustomize
 
+In the quickstart guide, we have demonstrated [how to install ironic with kustomize](../quick-start.md#deploy-ironic), by creating an ironic kustomization overlay.
```
Comment (tuminoid) [inline docs/user-guide/src/ironic/ironic_installation.md:65]: Wrap lines at 80, even though it is not forced here due legacy. Makes it easier for me to enable it later.

Comment (mquhuy) [reply docs/user-guide/src/ironic/ironic_installation.md]: Thanks. Does it look ok now?

Comment (tuminoid) [reply docs/user-guide/src/ironic/ironic_installation.md]: Much better, thank you!

**Review by lentzi90 — COMMENTED** [review]
/approve

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Pull request merged**